presto-i18n/src/main/java/com/facebook/presto/i18n/I18nMyanmarFunctions.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.i18n;
+
+import com.facebook.presto.spi.function.Description;
+import com.facebook.presto.spi.function.ScalarFunction;
+import com.facebook.presto.spi.function.SqlType;
+import com.facebook.presto.spi.type.StandardTypes;
+import com.google.myanmartools.TransliterateZ2U;
+import com.google.myanmartools.ZawgyiDetector;
+import io.airlift.slice.Slice;
+
+import static io.airlift.slice.Slices.utf8Slice;
+
+public final class I18nMyanmarFunctions
+{
+    private I18nMyanmarFunctions() {}
+
+    static ZawgyiDetector detector = new ZawgyiDetector();
+    static TransliterateZ2U z2uTransliterator = new TransliterateZ2U("Zawgyi to Unicode");
+
+    @Description("labels whether input strings use Unicode or Zawgyi font encoding")
+    @ScalarFunction
+    @SqlType(StandardTypes.VARCHAR)
+    public static Slice myanmarFontEncoding(@SqlType(StandardTypes.VARCHAR) Slice slice)
+    {
+        if (detector.getZawgyiProbability(slice.toStringUtf8()) > 0.9) {
```

Comment:
is 0.9 always a good threshold?

Reply:
This is what we use for the Facebook site - we are experimenting with this, but for now we don't have data supporting a different threshold.

Reply:
Maybe define it as a constant. You can also consider adding a function taking threshold as input.

Reply:
Added a constant - I think adding functions w. threshold as input is a great idea, but let's keep it simple in this rev.

I'm thinking of:
- add threshold as an option to both `MYANMAR_FONT_ENCODING` and `MYANMAR_NORMALIZE_UNICODE`
- add an option for whether to split on newlines to `MYANMAR_FONT_ENCODING` - a lot of times, Myanmar content will be posted as paragraphs of Zawgyi followed by paragraphs of Unicode

but for now, `MYANAMR_NORMALIZE_UNICODE` should help the most with any text matching queries